packages/block-library/src/navigation/index.php

```diff
@@ -446,8 +447,8 @@ function block_core_navigation_get_default_pages_fallback() {
 	$wp_insert_post_result = wp_insert_post(
 		array(
 			'post_content' => $default_blocks,
-			'post_title'   => 'Navigation', // TODO - use the template slug in future.
-			'post_name'    => 'Navigation', // TODO - use the template slug in future.
+			'post_title'   => 'Navigation',
```

Comment:
Translatable?

Reply:
I pulled out a separate PR in case this one doesn't get merged https://github.com/WordPress/gutenberg/pull/48773